lib/model/content.dart

```diff
@@ -717,6 +721,10 @@ class UserMentionNode extends InlineContainerNode {
   //   final bool isSilent; // TODO(#647)
 }
 
+class TopicMentionNode extends MentionNode {
```

Comment:
I think the name "topic mention" for this, with no further context, will be confusing — it sounds like it might refer to the `#**channel > topic**` feature, which is unrelated.